src/Uno.Foundation/Point.cs

```diff
@@ -10,6 +11,7 @@
 
 namespace Windows.Foundation
 {
+	[DebuggerDisplay("{X},{Y}")]
```

Comment:
Kind of useless since the `ToString` already does something similar

Reply:
Nope it's not working everywhere in VS. The `DebuggerDisplay`, yes.